na-toolbar.html

```diff
@@ -49,8 +49,12 @@
     </style>
     <iron-icon icon="bookmark"></iron-icon>
     <h1>Notes</h1>
-    <iron-icon icon="cloud" hidden$="[[!online]]"></iron-icon>
-    <iron-icon icon="cloud-off" hidden$="[[online]]"></iron-icon>
+    <span hidden$="[[!online]]">
+      <iron-icon icon="cloud"></iron-icon>
+    </span>
+    <span hidden$="[[online]]">
+      <iron-icon icon="cloud-off"></iron-icon>
+    </span>
```

Comment:
Not sure I understand why this is a slippery slope -- a lot of users (mistakenly) expect hidden to work a certain way, and we are helping make the element easier to use by meeting that expectation. This is no different than accepting feature requests.